src/components/Sections/Lessons/tabs/Series/Container.js

```diff
@@ -96,11 +96,11 @@ const SeriesContainer = ({ t }) => {
 
   const getHeader = node => node.id === "byTopics"
     ? t(`lessons.tabs.seriesTree.byTopics`)
-    : `${t(`lessons.tabs.seriesTree.bySourcesOf`)}${node.name}`;
+    : `${t(`lessons.tabs.seriesTree.bySourcesOf`)}${` `}${node.name}`;
 
   const renderKabbalist = node => {
     if (node?.children?.length > 0) {
-      const initLevel = 2;
+      const initLevel = 3;
```

Comment:
Can you pls add a comment what does it mean?

Reply:
it is used in the header in 2 rows